spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -30,19 +30,20 @@
 
     # check whether the server was associated with the vm
     server = @ems_hawkular.middleware_servers.first
-    expect(server.lives_on_id).to eql(@vm.id)
-    expect(server.lives_on_type).to eql(@vm.type)
+    # TODO: Restore these expectations
```

Comment:
> They should appear if the hawkular-services are deployed on a virtual machine managed by RHEV provider and this provider is also added to MiQ. 

That's correct.

> The cross-linking should just happen automatically.

Depending on the definition of automatically, that's also correct :) @abonas can confirm how it works on the UI, but on this side, all we need to do is to set the `lives_on` field. Something else will compare these values and show that accordingly on the UI.

> If I recall correctly the https://hawkular.torii.gva.redhat.com is running on RHEV and both the hawkular and RHEV can be added as providers to MiQ (juca can give you the access).

Also correct.

> Also, do you have any idea how to workaround the ^ for containers? Perhaps we can change the permissions of /etc/machine-id so that it's writable by non-root guys, right? That would work. If you are not against it I am willing to add it back.

I have no idea about that, though, as it probably depends on the outcome of the discussions about the machine ID for containers. In my *opinion*, `/etc/machine-id` should not contain the hostname, or anything that doesn't conform with the spec that rules this file: https://www.freedesktop.org/software/systemd/man/machine-id.html